### ISSUE TITLE
Cant' access the cache-update with the following issue

### ISSUE THREAD

**Issue by firmanpolyrific** [enteresanlikk/amp-cache-update#1]
Your client does not have permission to get URL /update-cache/c/s/nerdist.com/article/the-boys-homelander-america-hero-trump/%3Famp/article?amp_action=flush&amp_ts=1565064482&amp_url_signature=pdItYu8m6nEe8j-tLKKU7Nne6TnnD-TrE7Nxu816jDTFCSyWFjmBvdXZVGe77ojboBT5kbm9e4esT6fctvmICpsfh0SL8MIHDavEY33lmv6B56LmdLz9hrPLLLlOQj6h1IhQKutNiEyaDzdOgZy7uQTXXgggbMwliYd-c87117WxAbAj3eaO5X3l2_cEr-XtbYTe4WrP8mW7GJXX7tPwNo9ZbrUSkjhV7O6LV-mjyrNVz7sy8Um45usgQV3h6a1A64JuIuua5fJUXWi4zzv_rvpH0bWzThvuMuTDJzZg8BO22kzJoE2SEt_cvKzHYYpj2MqSOxIVnpcpLErxQr_HZA from this server. (Client IP address: 36.81.169.173)

Invalid public key due to ingestion error: Invalid Content That’s all we know.

can you show me the correct way to generate .pem file? it's a bit confusing if I read the docs from here, do the public key has to renamed to apikey.pub before upload? (on step 2)

https://developers.google.com/amp/cache/update-cache#rsa-keys

and also where we generate the rsa key, on our local or online server?

**Comment by firmanpolyrific** [comment]
any update on this?

**Comment by atakancelik** [comment]
Hello, sorry for the late reply. Can you control the data path you created? Do you have access when you go to the URL? If you wish, you can read the article on "https://medium.com/@atakancelik/amp-cache-update-afd15f98a45a" and check your actions.

**Comment by firmanpolyrific** [comment]
hi @atakancelik 

I think I have followed the article already, I need to manually update the google amp cache.

https://developers.google.com/amp/cache/update-cache#rsa-keys
when I following this doc, it seems not pretty clear about the steps,

does the .pem is generated on my server or it on my local?
and how we can convert public-key.pem to apikey.pub, are we just rename it? 
I tried to use ssh-agent to generate .pub as well but it seems not the correct way

and also for note: I already disallow robot.txt to crawl the apikey.pub, change permission to 777 for the apikey.pub but still no luck

any help will be appreciated for this :)
